tests/tests_pytorch/utilities/test_parsing.py

```diff
@@ -142,7 +161,17 @@ def test_lightning_getattr():
 
 def test_lightning_setattr(tmpdir):
     """Test that the lightning_setattr works in all cases."""
-    models = model_cases()
+    *models, trainer1, trainer2 = (
+        model1,
+        model2,
+        model3,
+        model4,
+        model5,
+        model6,
+        model7,
+        model8,
+        *trainers,
+    ) = model_and_trianer_cases()
```

Comment:
I think this would look nicer (formatting needs to be fixed)

```suggestion
        model1,
        model2,
        model3,
        model4,
        model5,
        model6,
        model7,
        model8,
        *trainers,
    = *model_and_trianer_cases()
```

Reply:
It would, but you wouldn't get `models` variable, which is being used in the tests. Plus, I'm fairly sure `black` would have something to say about it

Reply:
feel free to modify the test, or resolve this suggestion.

Reply:
Refactored